ocp_install_env.sh

```diff
@@ -36,14 +36,39 @@ function extract_installer() {
     extract_command openshift-baremetal-install "$1" "$2"
 }
 
+function save_release_info() {
+    local release_image
+    local outdir
+
+    release_image="$1"
+    outdir="$2"
+
+    oc adm release info --registry-config "$PULL_SECRET_FILE" "$release_image" -o json > ${outdir}/release_info.json
+}
+
+# Gives e.g 4.7.0-0.nightly-2020-10-27-051128
+function openshift_release_version() {
+    jq -r ".metadata.version" ${OCP_DIR}/release_info.json
+}
+
+# Gives us e.g 4.7 because although OPENSHIFT_VERSION is set by users,
+# but is not set in CI
+function openshift_version() {
+    jq -r ".metadata.version" ${OCP_DIR}/release_info.json | head -c3
```

Comment:
Probably not very urgent, but what about instead using something like:

`jq -r ".metadata.version" ${OCP_DIR}/release_info.json | grep -oP "\d\.\d+"` ?

It will avoid report a wrong number when version will be > 4.9

Reply:
Good point, will do!